governance/polls/templates/Collateral Onboarding/Proposal - Add $collateral as a PSM - $date_MONTH_DD,_YYYY.md

```diff
@@ -28,11 +28,11 @@ The community can vote in this poll to express support or opposition to the onbo
 
 **Proposed $collateral PSM Parameters**
 
-* [Fee In](https://makerdao.world/en/learn/governance/module-psm) (`tin`): $tin
-* [Fee Out](https://makerdao.world/en/learn/governance/module-psm) (`tout`): $tout
-* [Maximum Debt Ceiling](https://makerdao.world/en/learn/governance/module-dciam) (`line`): $line
-* [Target Available Debt](https://makerdao.world/en/learn/governance/module-dciam) (`gap`): $gap
-* [Ceiling Increase Cooldown](https://makerdao.world/en/learn/governance/module-dciam) (`ttl`): $ttl
+* [Fee In](https://manual.makerdao.com/module-index/module-psm) (`tin`): $tin
```

Comment:
Ah, so one big advantage with the new manual is that we can link directly to headings. We should do this for the module docs especially!

Reply:
I am not sure I understand this, @LongForWisdom